### PR TITLE
Trigger Pubmed workflow and check FTP results

### PR DESCRIPTION
**Pull request by giorgiosironi**

### CODE REVIEW

File: spectrum/checks.py [hunk 1]
```diff
@@ -867,11 +867,18 @@ def _build_url(path, host):
...
     'articles/{id}/elife-{id}-v{version}.pdf'
 )
-PACKAGING_BUCKET = BucketFileCheck(
+PACKAGING_BUCKET_OUTBOX = BucketFileCheck(
     aws.S3,
     SETTINGS['bucket_packaging'],
-    '{vendor}/{folder}/elife{id}.xml',
-    '{vendor}/{folder}/elife{id}.xml'
+    '{vendor}/outbox/elife{id}.xml',
+    '{vendor}/outbox/elife{id}.xml'
```
Comment (gnott) [inline spectrum/checks.py]: This spectrum code is all new to me. Should / could this last argument, which is maybe the S3 prefix, be ``'{vendor}/outbox/'``?

File: spectrum/test_article.py [hunk 2]
```diff
@@ -172,11 +172,14 @@ def test_adding_article_fragment(generate_article, modify_article):
...
     _ingest_and_publish_and_wait_for_published(article)
-    checks.PACKAGING_BUCKET.of(vendor="pubmed", folder="outbox", id=article.id())
-    #input.BOT_WORKFLOWS.start_pubmed()
-    #python econtools/econ_article_feeder.py -p elife-00666-vor-r1.zip -r 1  ct-elife-production-final ct-workflow-starter-queue PubmedArticleDeposit
-    #checks.PACKAGING_BUCKET.published("pubmed", "20180112", article.id())
+    checks.PACKAGING_BUCKET_OUTBOX.of(vendor="pubmed", folder="outbox", id=article.id())
+    input.BOT_WORKFLOWS.pubmed()
+
+    (xml, ) = checks.PACKAGING_BUCKET_BATCH.of(vendor="pubmed", last_modified_after=test_start)
+    checks.PUBMED.of(xml=xml)
```
Comment (gnott) [inline spectrum/test_article.py]: :+1:

File: spectrum/checks.py (hunk 1 above)
Comment (giorgiosironi) [reply spectrum/checks.py]: The last argument is a prefix used to cut down on the number of files listed from S3 when the main expression has to be matched. `{vendor}` and `{id}` are substituted when `of()` is called, so in this case we can essentially use the whole file name as the prefix, which is very efficient. In other cases we don't know the file name in advance, so we have to stop the prefix at the "folder" level.

Comment (gnott) [reply spectrum/checks.py]: Ah yes, the published folder subfolders vary by the date it is run, gotcha - thanks!

**Pull request merged**